src/plugins/vis_augmenter/public/utils/utils.test.ts

```diff
@@ -16,26 +16,150 @@ import {
   getMockAugmentVisSavedObjectClient,
   generateAugmentVisSavedObject,
 } from '../saved_augment_vis';
+import { AggConfigs, AggTypesRegistryStart } from '../../../data/common/search/aggs';
+import { stubIndexPatternWithFields } from '../../../data/common/index_patterns/index_pattern.stub';
+import { IndexPattern } from '../../../data/common/index_patterns/index_patterns';
+import { mockAggTypesRegistry } from '../../../data/common/search/aggs/test_helpers';
 
 describe('utils', () => {
-  // TODO: redo / update this test suite when eligibility is finalized.
-  // Tracked in https://github.com/opensearch-project/OpenSearch-Dashboards/issues/3268
   describe('isEligibleForVisLayers', () => {
-    it('vis is ineligible with invalid type', async () => {
+    const validDimensions = {
+      x: {
+        params: {
+          bounds: {
+            min: '2023-03-22T21:55:12.455Z',
+            max: '2023-03-23T21:55:12.455Z',
+          },
+        },
+        label: 'order_date per 30 minutes',
+      },
+    } as VislibDimensions;
+    const configStates = [
+      {
+        enabled: true,
+        type: 'histogram',
+        params: {},
+      },
+      {
+        enabled: true,
+        type: 'date_histogram',
+        params: {},
+      },
+    ];
```

Comment:
why is there both a `histogram` and a `date_histogram` here?

Reply:
Removed histogram and used other types that made more sense.